docs/readthedocs/conf.py

```diff
@@ -18,12 +18,12 @@
 # -- Project information -----------------------------------------------------
 
 project = 'PowerSystemDataModel'
-copyright = u'2020. TU Dortmund University, Institute of Energy Systems, Energy Efficiency and Energy Economics, Research group Distribution grid planning and operation '
+copyright = u'2023. TU Dortmund University, Institute of Energy Systems, Energy Efficiency and Energy Economics, Research group Distribution grid planning and operation '
 author = 'Johannes Hiry, Debopama Sen Sarma, Chris Kittl'
 
 # The full version, including alpha/beta/rc tags
-version = '1.0'
-release = '1.0.1-SNAPSHOT'
+version = '2.1.0'
+release = '3.0-SNAPSHOT'
```

Comment:
Please check out the [documentation for version and release](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version)

Reply:
@staudtMarius that's not it, yet. `version` and `release` refer to the same version number, but `version` is shorter (`3.0`).